test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -1073,10 +1095,11 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
                 {
                     await connection.Send(
                         "POST / HTTP/1.1",
+                        "Host:",
                         "Transfer-Encoding: chunked",
                         "",
                         "gg");
-                    await responseWritten.WaitAsync();
+                    await responseWritten.WaitAsync().TimeoutAfter(TimeSpan.FromSeconds(10));
```

Comment:
other than my impatience, no. I don't think we're consistent in which timeout we use.